api/resources/user_quests.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+import json
+from flask import jsonify
+
+import bleach
+# import jsons
```

Comment:
Could we clean up the unneeded commented out code in this file?

Reply:
^^^